modular_zubbers/code/modules/loadout/categories/~donator/donator_personal.dm

```diff
@@ -104,6 +104,11 @@
     item_path = /obj/item/clothing/suit/misc/holographic
     ckeywhitelist = list("blovy")
 
+/datum/loadout_item/suit/skyymed_jacket // donator item for LT3
+	name = "Expedition Medical Jacket"
+	item_path = /obj/item/clothing/suit/toggle/labcoat/skyrat/skyymed
+	ckeywhitelist = list("lt3", "gavla")
+	restricted_roles = list (JOB_MEDICAL_DOCTOR, JOB_SECURITY_MEDIC, JOB_CHEMIST, JOB_PARAMEDIC, JOB_ORDERLY, JOB_CHIEF_MEDICAL_OFFICER)
```

Comment:
I asked Zender about the ckey whitelist since we've stopped accepting them, but he allowed it as a special exception. But I'm going to be rather strict regarding the stats, you have two options since donor items need to remain equal.

1. You can further restrict the roles to only security medic and orderly that already start with this armor and ```allowed +=``` item list. Yes, CMO is included because CMO does not have armor round start.

```suggestion
	restricted_roles = list (JOB_SECURITY_MEDIC, JOB_ORDERLY)
```

2. You can remove the`` allowed +=```  bit of code **and** give it ```/datum/armor/toggle_labcoat```

```
/obj/item/clothing/suit/toggle/labcoat/skyrat/skyymed/Initialize(mapload)
	. = ..()
	allowed += list(/obj/item/gun, /obj/item/melee/baton/telescopic)```

Reply:
I'll just make it a normal labcoat.

Reply:
> I'll just make it a normal labcoat.

Alright! Sounds great, otherwise no more issues from me.

Reply:
Yep, done!